conf/script/build-ha-io

```diff
@@ -144,8 +144,8 @@ if [[ -f $argsfile ]]; then
            ip1)          ip1=$value     ;;
            ip2)          ip2=$value     ;;
            interface)    iface=$value   ;;
-           left_iface)   left_iface=$value   ;;
-           right_iface)  right_iface=$value   ;;
+           left-iface)   left_iface=$value   ;;
+           right-iface)  right_iface=$value   ;;
```

Comment:
Do we need to change at other places too in this file? @andriytk

Reply:
Like where?

Reply:
sorry, never mind

Reply:
@andriytk 
We need change on line: https://github.com/Seagate/cortx-ha/pull/159/files#diff-7801cbb90727f0c5f7c79eac586ff9c6R96

Reply:
That's taken care of, just checked the code.
We are good.